timezone_converter/comparison_view.py

```diff
@@ -59,24 +60,19 @@ def _build_table(self) -> Table:
         for header in headers:
             table.add_column(header, justify='center')
 
-        hour_range = range(24)
+        hours_to_print: Iterable[int] = range(24)
```

Comment:
This type annotation is needed to make mypy happy

Reply:
Interesting, I actually had the same idea but I only played with it for a couple of minutes and could not make mypy happy, but seems like I not tickling it the right way.